less/variables.less

```diff
@@ -24,3 +24,4 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@phone: (max-width: 500px);
```

Comment:
change syntax of phone to: @phone: ~ "(max-width: 500px)" using escaping and converts to string